client/signup/jetpack-connect/authorize-form.jsx

```diff
@@ -639,23 +676,28 @@ const JetpackConnectAuthorizeForm = React.createClass( {
 
 export default connect(
 	state => {
-		const remoteSite = getAuthorizationRemoteSite( state );
-
+		const remoteSiteUrl = getAuthorizationRemoteSiteUrl( state );
+		const siteSlug = withoutHttp( remoteSiteUrl ).replace( /\//g, '::' );
```

Comment:
We should probably replace `/` with '::` (or the opposite) consistently within the selectors and helper functions, and not directly here.